x-pack/solutions/security/plugins/security_solution/public/plugin.tsx

```diff
@@ -203,6 +205,7 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
       id: 'securityAiAssistantManagement',
       title: ASSISTANT_MANAGEMENT_TITLE,
       hideFromSidebar: true,
+      hideFromGlobalSearch: !this.isServerless,
```

Comment:
I could see the AI Assistant in the global search bar at the moment in standalone ESS + Kibana and serverless projects. Could you please double check?

<img width="1509" height="858" alt="Screenshot 2025-10-22 at 15 50 26" src="https://github.com/user-attachments/assets/8f84e3db-995f-4563-b1ef-b21dea112d1d" />
<img width="1512" height="818" alt="Screenshot 2025-10-22 at 15 51 20" src="https://github.com/user-attachments/assets/ccd46957-b396-45a4-ad62-db6a36859769" />
<img width="2560" height="1244" alt="Screenshot 2025-10-22 at 15 59 58" src="https://github.com/user-attachments/assets/0239507a-fb7d-48e1-a0ff-dc8d4ff8d91d" />